docs/src/diagnostics/developers_diagnostics.md

```diff
@@ -107,6 +107,7 @@ hourly_averages(FT, short_names...; output_writer) = common_diagnostics(
     60 * 60 * one(FT),
     (+),
     output_writer,
+    nothing,
```

Comment:
```suggestion
    nothing, # start_date
```
I think it's helpful to add a comment explaining which argument `nothing` is being passed for, but this is personal choice